tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import
+
+from exam import fixture
+
+from sentry.api.serializers import serialize
+from sentry.incidents.logic import create_alert_rule
+from sentry.incidents.models import (
+    AlertRuleAggregations,
+    AlertRuleThresholdType,
+)
+from sentry.testutils import APITestCase
+
+
+class AlertRuleListEndpointTest(APITestCase):
```

Comment:
Should this be AlertRuleDetailsEndpointTest?

Reply:
😬